Dockerfile

```diff
@@ -599,7 +600,8 @@ COPY --from=vpnmanager-builder --chown=nextjs:nodejs /workspace/apps/vpnmanager/
 # https://nextjs.org/docs/advanced-features/output-file-tracing
 COPY --from=vpnmanager-builder --chown=nextjs:nodejs /workspace/apps/vpnmanager/.next/standalone ./apps/vpnmanager
 COPY --from=vpnmanager-builder --chown=nextjs:nodejs /workspace/apps/vpnmanager/.next/static ./apps/vpnmanager/.next/static
-
+COPY --from=vpnmanager-builder --chown=nextjs:nodejs /workspace/apps/vpnmanager/scripts ./apps/vpnmanager/scripts
+COPY --from=vpnmanager-builder --chown=nextjs:nodejs /workspace/apps/vpnmanager/app.json ./app.json
```

Comment:
This should be in contrib/dokku folder (I'm assuming it's only used by Dokku)

Reply:
@kilemensi `app.json` should be at the root for the scripts to be executed

Reply:
Location of `app.json` is [configurable](https://dokku.com/docs/advanced-usage/deployment-tasks/?h=app.json+location#changing-the-appjson-location) @koechkevin